lint_and_test.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -u
+
+checks_to_run=${1:-"everything"}
+shift || true
+
+
+lint() {
+    echo "⛄️ flake8 check..."
+    python3 -m flake8 src tests
```

Comment:
what about `isort` and `autopep` tests?

Reply:
`flake8` checks PEP8 compliance, so it should lint according to the same rules as `autopep` is formatting, and I added the `flake8-isort` plugin instead of using `isort` directly, it's easier to integrate with VSCode that way.